simpl/utils/importing.py

```diff
@@ -0,0 +1,85 @@
+"""Utilities for importing.
+
+The function import_me_maybe() uses the signature
+of the importlib.import_module() function.
+
+Usage:
+
+    from simpl.utils import import_me_maybe
+
+    eventlet = import_me_maybe('eventlet')
+    # this issues a warning (only once) if
+    # eventlet is not found
+
+    # to prevent warnings from ever occurring
+    from simpl.utils import disable_warnings_for
+
+    # this disables the corresponding warnings globally
+    disable_warnings_for('eventlet')
+
+    # this could be in another module
+
+    eventlet = import_me_maybe('eventlet')
+    # produces no warning now
+"""
+import importlib
+import inspect
+import os
+import warnings
+
+from simpl.exceptions import DependencyRequiredWarning
+
+
+def import_me_maybe(name, package=None):
```

Comment:
The 'default' (once) warnings filter will warn once based on the origin of the warning. So I would assume that a different reason means different line of code where `import_me_maybe` is called, so you would get both warnings.